frontend/src/index.css

```diff
@@ -115,3 +115,93 @@
         opacity: 1;
     }
 }
+
+/* ENTIRE HOUSE */
```

Comment:
if you dont want to manage css, you can also express this using styled components or scss/less and then compile down using webpack